plugins/woocommerce-admin/webpack.config.js

```diff
@@ -165,7 +166,14 @@ const webpackConfig = {
 
 		// We reuse this Webpack setup for Storybook, where we need to disable dependency extraction.
 		! process.env.STORYBOOK &&
-			new WooCommerceDependencyExtractionWebpackPlugin(),
+			new WooCommerceDependencyExtractionWebpackPlugin( {
+				requestToExternal( request ) {
+					if ( request === '@wordpress/components/build/ui' ) {
```

Comment:
We have three different Spinner components:

- [`@woocommerce/components/spinner`](https://woocommerce.github.io/woocommerce-admin/#/components/)
- [`@wordpress/components/spinner`](https://wordpress.github.io/gutenberg/?path=/story/components-spinner--default)
- [`@wordpress/components/ui/spinner`](https://wordpress.github.io/gutenberg/?path=/story/components-experimental-spinner--default)

And we need to use the one from `@wordpress/components/ui/spinner`, but it's not included in external wp.components.